base/docs/helpdb/Base.jl

```diff
@@ -2082,7 +2082,11 @@ kron
 """
     >>(x, n)
 
-Right bit shift operator, preserving the sign of `x`.
+Signed right bit shift operator, `x >> n`. The result is `x` shifted right by `n` bits,
```

Comment:
`>>` is not always signed; it is signed for signed types, and unsigned for unsigned types.

That is, unsigned types always shift in zeros from the left.

The result is not undefined for negative `n`; there is generally very little undefined behaviour in Julia. Instead, `n` is reinterpreted as unsigned number. (This could/should/will change, leading to an error instead in this case.)